arch/arm/soc/nxp_lpc/lpc54xxx/soc.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Board configuration macros for the nxp_lpc54114 platform
+ *
+ * This header file is used to specify and describe board-level aspects for the
+ * 'nxp_lpc54114' platform.
+ */
+
+#ifndef _SOC__H_
+#define _SOC__H_
+#include <misc/util.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef _ASMLANGUAGE
+#include <device.h>
+#include <misc/util.h>
+#include <drivers/rand32.h>
+#include <fsl_common.h>
+#endif /* !_ASMLANGUAGE */
+
+#ifdef __cplusplus
+}
+#endif
+
+#define BOARD_BOOTCLOCKFROHF48M_CORE_CLOCK		48000000U
```

Comment:
It is the main clock source frequency of FRO which later gets divided as core clock,peripheral clock and bridge clock.
Since I am using FRO three are three possible values . As of now it is only being used in soc.c but I am not sure if it will or not be used in other libraries of arch inn future . That is why I have put it here in soc.h.

Please let me know if I still need to change it.